lesson_7/edmond_sabou/lesson7_3.py

```diff
@@ -0,0 +1,31 @@
+print_registry = []
+
+
+def register(func):
+
+    def appender(*args):
```

Comment:
This code can be further simplified; is the inner function really needed?
Justify the answer, whichever that might be

Reply:
now that you say that it works with a single function, I tried and it worked. 
can't say unfortunately why this works and others don't